### ISSUE TITLE
Meaningful/content SVG elements should have role="img" attribute

### ISSUE THREAD

**Issue by danchamp** [localgovdrupal/localgov_base#109]
[Accessibility audit: issue 58](https://docs.google.com/spreadsheets/d/1t8rRIQe_1kogxPBZDKcsbf8YxtEfC9iGJvidkagNSXY/edit#gid=1779042480&range=E59)

I don't think the theme features any non-decorative SVGs, but requires review.

**Comment by danchamp** [comment]
There are no meaningful / content `<svg>` elements in the theme. `logo.svg` is the only possible candidate, and it's used in an `<img>` element.
